tools/run.js

```diff
@@ -35,34 +36,38 @@ module.exports = function(program) {
 
   function run(options) {
     console.log("RUN YOTE");
-    let runOptions = ["web","server","mobile"];
+    let runOptions = [];
     // let PascalName =utils.capitalizeFirstLetter(appName);
     const mobileProjectName = utils.getYoteMobileProjectName();
     console.log(mobileProjectName);
-    // console.log(options);
-    if(options.A || (!options.M && !options.S && !options.W)) {
-      console.log('run all the things');
-    } else if(options.W == undefined) {
-      runOptions.splice(runOptions.indexOf('web'), 1);
-    } else if(options.S == undefined) {
-      runOptions.splice(runOptions.indexOf('server'), 1);
-    } else if(options.M == undefined) {
-      runOptions.splice(runOptions.indexOf('mobile'), 1);
+    let tabCmd = options.T ? 'ttab' : 'ttab -w'; 
+    if(options.A || (!options.W && !options.S && !options.M)) {
+      runOptions = ["web","server","mobile"];
+      tabCmd = 'ttab'; 
```

Comment:
wont setting tabCmd here this override the above?

Reply:
@grantfowler yes, so yote run (with no flags) will default to tabs and not windows